.pre-commit-config.yaml

```diff
@@ -73,7 +73,7 @@ repos:
     rev: v0.800
     hooks:
     - id: mypy
-      additional_dependencies: [numpy>=1.20]
+      additional_dependencies: ["numpy>=1.20,<1.21"]
```

Comment:
numpy 1.21 made `ndarray` generic, so we will need to update to fix that. I'll do those changes in #30